iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -1,11 +1,68 @@
 from rest_framework import serializers
 
 from iaso.models import OrgUnitChangeRequest
+from collections import OrderedDict
+
+from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.geos.error import GEOSException
+from django.utils.translation import gettext_lazy as _
+
+
+class PointFieldWithAccuracy(serializers.Serializer):
+    """
+    Expected input:
+    {
+        "latitude": 49.8782482189424,
+        "longitude": 24.452545489,
+        "altitude": 5.1,
+        "accuracy": 1.2
+    }
+    """
+
+    default_error_messages = {
+        "invalid": _("Enter a valid location."),
+    }
+
+    point_field: str = "new_location"
+    accuracy_field: str = "new_accuracy"
+
+    def __init(self, point_field="new_location", accuracy_field="new_accuracy"):
```

Comment:
Making `point_field` and `accuracy_field` mandatory would probably make it more explicit.